agent_server.py

```
@@ -1499,11 +1499,27 @@ async def startup():
     Modules.throttled_logger = ThrottledLogger(logger, interval=30.0)
     _rewire_computer_use_dependents()
+
+    async def _init_browser_use_background():
+        try:
+            bu = await asyncio.to_thread(BrowserUseAdapter)
+            Modules.browser_use = bu
+            Modules.task_executor.browser_use = bu
+            logger.info("[Agent] BrowserUseAdapter ready (background init)")
+            try:
+                await _fire_agent_llm_connectivity_check()
+            except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)